psc-bundle/README.md

```diff
@@ -4,14 +4,15 @@ A dead code elimination tool for PureScript-style CommonJS modules. This can be
 
 ## Usage
 
-    psc-bundle FILE (-m|--module ARG) [--main ARG] [--namespace ARG]
-    
+    psc-bundle FILE (-m|--module ARG) [--main ARG] [--namespace ARG] [--optimize uncurry]
```

Comment:
Thanks, fixed it. It was an explicit wish to have single options for any optimization.  However, it is easy to change this if desired.